core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -76,6 +76,7 @@ public static LocalCacheManager create(AlluxioConfiguration conf) throws IOExcep
     PageStore pageStore = PageStore.create(conf);
     try {
       Collection<PageInfo> pageInfos = pageStore.getPages();
+      LOG.info("Loading {} existing pages", pageInfos.size());
```

Comment:
Should this be something like "Creating LocalCacheManager with {} existing pages" ?